lib/go-atscfg/strategiesdotconfig.go

```diff
@@ -162,11 +164,18 @@ func getStrategyErrorCodes(codes []int) string {
 
 func getStrategyGroups(svc *ParentAbstractionService) string {
 	txt := ""
-	if len(svc.Parents) != 0 {
-		txt += "\n" + `      - *group_parents_` + svc.Name
-	}
-	if len(svc.SecondaryParents) != 0 {
-		txt += "\n" + `      - *group_secondary_parents_` + svc.Name
+	if getStrategySecondaryMode(svc.SecondaryMode) == "peering_ring" {
+		txt += "\n" + `      - *peers_group`
+		if len(svc.Parents) != 0 {
```

Comment:
Nitpick: the `group_parents_` is duplicated in both cases, it could be de-duplicated and moved outside the if-statement